src/transform/tsTransform.ts

```diff
@@ -75,12 +73,28 @@ export function tsTransform(props: ITypescriptTransformProps): ts.TranspileOutpu
   }
 
   const after = [];
+
   if (props.replacements || props.webWorkers) {
     after.push(moduleTransformer());
   }
+
   return ts.transpileModule(props.input, {
     fileName: props.fileName,
     compilerOptions: props.compilerOptions,
-    transformers: { after: after },
+    transformers: {
+
+      // merge in any custom transformers (user-provided)
+      ...props.transformers,
```

Comment:
I think that spread operator is very expensive to be applied each time on every single file. Unless you convince me there is no difference performance wise ;-)

Reply:
Yeah you're right, it can be up to 3x slower than Array.concat for the case of the Array spread. But according to https://thecodebarbarian.com/object-assign-vs-object-spread.html Object spread operator is faster than Object.assign. I have no evidence on that tho. 

But, tbh. this is loaded only once and not executed every time/at bundle time. How fast do you think, Array.concat or the object merge would be on this? I guess all the operations are waaay below 1 ms in combination. 

Also, for an alternative impl., we would need a deep merge here to have the same outcome. For a good deep merge, we'd need loops or, even worse, external libs like lodash. TypeScript can generate these loops for us by using the ... spread on the object literal. I would always go for high performance for every operation inside of a build step, but in this specific case I would rather go for readability, less and more elegant code. What do you think?